added michelles advice

michellemidnight opened the pull request:
Some Advice

acodeinprogress commented:
Hey @michellemidnight,

I noticed that you didn't add any CSS but your advice page links to the main style.css file. If you could add in some CSS, that would be great! Once you do that, you can add, commit, and push your work and it will join your pull request here.

Let us know if you need any help!

xoxo,
Sharon

acodeinprogress commented:
Hey @michellemidnight,

It looks like we haven’t heard from you in a while. We’d love it if you would make the changes we requested above so we can get this all merged in! If no changes are made in 10 days, this pull request will be closed. 

Let us know if you need any help!

Happy git-ing,
Sharon

michellemidnight commented:
Sorry!! You can close the pull request. I had already deleted the files and moved to the next section.

Sent from my iPhone

> On Nov 18, 2017, at 12:07 PM, Sharon <notifications@github.com> wrote:
> 
> Hey @michellemidnight,
> 
> It looks like we haven’t heard from you in a while. We’d love it if you would make the changes we requested above so we can get this all merged in! If no changes are made in 10 days, this pull request will be closed.
> 
> Let us know if you need any help!
> 
> Happy git-ing,
> Sharon
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
